### PR TITLE
Lambda Beta Documentation

### PR DESCRIPTION
**Pull request by erichaberkorn**
This PR has already been reviewed in #13372 and #13245.

### CODE REVIEW

**Review by pglass — APPROVED** [review]

File: website/content/docs/lambda/registration.mdx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+The following tags are supported. In all cases, the `<PREFIX>` should be
+`serverless.consul.hashicorp.com/v1alpha1/lambda`. For example,
+`serverless.consul.hashicorp.com/v1alpha1/lambda/enabled`.
+
+| Tag | Description |
+| - | - |
+| `<PREFIX>/enabled` | Determines if Lambda registrator will sync the Lambda into Consul. |
+| `<PREFIX>/payload-passthrough` | Determines if the body Envoy receives is converted to JSON or directly passed to Lambda. This attribute is optional and defaults to `false`. |
+| `<PREFIX>/invocation-mode` | Specifies the [Lambda invocation mode](https://docs.aws.amazon.com/lambda/latest/operatorguide/invocation-modes.html) Consul uses to invoke the Lambda. This tag will default to `SYNCHRONOUS`, but also supports `ASYNCHRONOUS` invocations. |
```
Comment (trujillo-adam) [inline website/content/docs/lambda/registration.mdx:215]:
```suggestion
| `<PREFIX>/invocation-mode` | Specifies the [Lambda invocation mode](https://docs.aws.amazon.com/lambda/latest/operatorguide/invocation-modes.html) Consul uses to invoke the Lambda. The default is `SYNCHRONOUS`, but `ASYNCHRONOUS` invocations are also supported. |
```
Cleaner?

File: website/content/docs/lambda/registration.mdx [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+| `<PREFIX>/namespace` | <EnterpriseAlert inline />This tag specifies the Consul namespace the service will be registered in. If `enterprise` is enabled on Lambda registrator, this defaults to `default`. |
```
Comment (trujillo-adam) [inline website/content/docs/lambda/registration.mdx:216]:
```suggestion
| `<PREFIX>/namespace` | <EnterpriseAlert inline />Specifies the Consul namespace the service will be registered in. Default is `default` if `enterprise` is enabled. |
```
Cleaner?

File: website/content/docs/lambda/registration.mdx [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+| `<PREFIX>/partition` | <EnterpriseAlert inline />This tag specifies the Consul partition the service will be registered in. If `enterprise` is enabled on Lambda registrator, this defaults to `default`. |
```
Comment (trujillo-adam) [inline website/content/docs/lambda/registration.mdx:217]:
```suggestion
| `<PREFIX>/partition` | <EnterpriseAlert inline />Specifies the Consul partition the service will be registered in. Defaults is `default` if `enterprise` is enabled. |
```
Cleaner?

File: website/content/docs/lambda/registration.mdx [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+| `<PREFIX>/aliases` | This tag specifies a `+`-separated string of Lambda aliases that will be registered into Consul. If this is set to `dev+staging+prod`, the `dev`, `staging`, and `prod` aliases of the Lambda function will be registered into Consul. |
```
Comment (trujillo-adam) [inline website/content/docs/lambda/registration.mdx:218]:
```suggestion
| `<PREFIX>/aliases` | Specifies a `+`-separated string of Lambda aliases that will be registered into Consul. For example, if set to `dev+staging+prod`, the `dev`, `staging`, and `prod` aliases of the Lambda function will be registered into Consul. |
```

**Review by trujillo-adam — APPROVED** [review]
Looks good. Marking as approved, but I added a few nits, questions, and comments.

File: website/content/docs/lambda/registration.mdx [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+</ImageConfig>
+
+1. EventBridge invokes the Lambda registrator based on CloudTrail Lambda events or a schedule.
+2. Lambda registrator determines how to reconcile Lambda's control plane state
+   with Consul state and ensures they are in sync by registering, updating, and
+   deregistering Lambda services.
+
+### Deploy Lambda Registrator
+
+1. Create a Terraform configuration and specify the `lambda-registrator` module. In the following example, the Lambda registrator is deployed to `https://consul.example.com:8501`. Refer to the Lambda registrator module documentation for additional usage information:
```
[the remark this answers is not in the document]
Comment (erichaberkorn) [reply website/content/docs/lambda/registration.mdx]: Now that beta1 is published, I added a link.

**Pull request merged**